docs/03-tools/01-e2e.md

```
@@ -0,0 +1,11 @@
+---
+description: E2E tool description
+---
+
+## Overview
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)